src/renderer/components/ft-subscribe-button/ft-subscribe-button.js

```diff
@@ -25,70 +25,99 @@ export default defineComponent({
       type: String,
       required: true
     },
+    hideProfileDropdownToggle: {
+      type: Boolean,
+      default: false
+    },
     subscriptionCountText: {
       default: '',
       type: String,
       required: false
     }
   },
+  data: function () {
+    return {
+      isProfileDropdownOpen: false
+    }
+  },
   computed: {
+    profileInitials: function () {
+      return this.profileDisplayList.map((profile) => {
+        return profile?.name?.length > 0 ? Array.from(profile.name)[0].toUpperCase() : ''
+      })
+    },
+
     profileList: function () {
       return this.$store.getters.getProfileList
     },
 
+    /* sort by 'All Channels' -> active profile -> unsubscribed channels -> unsubscribed channels */
```

Comment:
I think you meant to write subscribed in this comment, instead of unsubscribed twice :)